apps/super-app/src/common/Theme.ts

```diff
@@ -30,6 +30,26 @@ const callHomeTheme = createTheme(
       h4: {
         fontWeight: 600,
       },
+      h5: {
+        fontSize: 20,
+        fontWeight: 700,
+        color: '#133844',
+        fontFamily: 'Hind',
+        lineHeight: '28px',
+      },
+      h6: {
+        fontSize: 16,
+        fontWeight: 600,
+        color: '#297B94',
+        fontFamily: 'Hind',
+        textDecoration: 'underline',
```

Comment:
I wonder if we'll want all `h6`s to be underlined. Let's keep it and see if this ages well.